Azure/azure-cli
# {AppService} `az functionapp devops-pipeline create`: Omit making unsupported call for Centauri function apps

**Related command**
`az functionapp devops-pipeline create`

**Description**
The `az functionapp devops-pipeline create` command calls the show_app function, which still makes calls to unsupported APIs for Centauri. We are making sure those unsupported APIs are not called for Centauri function apps.

**Testing Guide**
1. Create a function app
2. Create the Dev Ops pipeline for that function app

---

This checklist is used to make sure that common guidelines for a pull request are followed.
